### DISCUSSION TITLE
Project shows as Excluded Project on my Hacktoberfest profile

### DISCUSSION THREAD

**Discussion by dro-1** [avinash201199/DSA-Questions#428] [General]
Does anyone know the reason why this is happening

**Comment by nitinkumar30** [comment]
![image](https://user-images.githubusercontent.com/40369168/135927166-7ebb510b-6df4-406f-99e9-13313068f854.png)

Maybe you can get to know after hovering on i button as I've done !

**Comment by avinash201199** [comment]
Thank you for your information :)
the project was excluded because there was so much pr at a time so the
organizer has taken it as a spam.
